apps/indexer/lib/indexer/fetcher/celo_epoch_rewards.ex

```diff
@@ -54,7 +54,7 @@ defmodule Indexer.Fetcher.CeloEpochRewards do
   def fetch_from_blockchain(blocks) do
     blocks
     |> Enum.map(fn block ->
-      case AccountReader.validator_group_reward_data(block) do
+      case AccountReader.epoch_reward_data(block) do
```

Comment:
is this fetching all epoch reward data or just voter rewards?

Reply:
all epoch rewards. it was a mistake from a previous PR